content/review/user-management.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "User Management"
+metaTitle: "sensenet - User Management"
+metaDescription: "sensenet User Management"
+---
+
+User management is a core function of the business, that includes user creation and assigning roles to the users. Effective user management contains adjusting the access rights of the users who change role, get promoted or leave the company. It is important to close these changes in short time, this way we can increase employee productivity.
+
+# Users and groups as content
+
+sensenet is built around content, so they are also users ang groups. Every content is built up by fields, so for example a user as content has a name field and password field (among others). The type of the content is called content type, and it defines a set of fields a content possesses, and also the behavior of it.
+
+# Built-in groups and users
+
+sensenet offers special and built-in groups and users by default. All users and almost all groups are stored in the /Root/IMS folder. It is possible jto create groups outside the IMS folder. These are local groups created under workspaces.
+
+Built-in groups in sensenet are regular groups, meaning their membership can be freely modified. They are used in the default Content Repository structure extensively, but no hard-coded logic is built on them (except for the Administrators group, please do not delete that one). This means you can freely change or replace these groups with your own custom groups in your permission settings. The best practice however is that you build on them, using the suggested purpose in the following sections.
+
+- public domain (/Root/IMS/Public) for the users
+
+- builtin domain (hidden) for technical users and groups that cannot be modified but its capabilities could be used in development
+
+   - Visitor
+   - Everyone
```

Comment:
```suggestion
   - Everyone: This is the largest group in the system: every single user is considered as a member of the Everyone group, except the Visitor user. This behavior cannot be changed. Please set permissions for this group carefully, as every logged in user will have access to those content. Note that you cannot set deny permissions for the Everyone group, because it would make that content inaccessible for - well, for everyone, including you.
```